model/common/tests/stencil_tests/test_cell_2_edge_interpolation.py

```diff
@@ -37,9 +40,10 @@ def reference(
         )
 
     @pytest.fixture
-    def input_data(self, grid):
+    def input_data(self, grid: base.BaseGrid) -> dict:
         if grid.get_offset_provider("E2C").has_skip_values:
             pytest.xfail("Stencil does not support missing neighbors.")
```

Comment:
I am confused, I thought we replaced all xfails by markers.

Reply:
Looks like these are all stencil tests from `common` maybe those were forgotten.